cfg/dist.js

```diff
@@ -1,42 +0,0 @@
-'use strict';
```

Comment:
@pjazdzewski1990 it should be provided by webpak 

basic we pass node to webpack config  
```js
//  build consts 
const buildTimestamp = Date.now();
const nodeEnv = process.env.NODE_ENV;
const isDevelopmentMode = nodeEnv !== 'production';
const uiBaseUrl = process.env.UI_BASE_URL || '';
```
then later in plugins we push 
```js
new webpack.DefinePlugin({
  // values are treated as code fragments so add quotes inside the value
  //FIXME don't replace whole 'process.env' object but individual properties
  'process.env': _.mapValues({
    IS_DEVELOPMENT_MODE: isDevelopmentMode,
    UI_BASE_URL: uiBaseUrl,
    BUILD_TIMESTAMP: buildTimestamp,
    NODE_ENV: nodeEnv // used by React for example
    // etc... 
  }, v => JSON.stringify(v))
}),
```